main.py

```diff
@@ -1,12 +1,18 @@
 """ToDo."""
-from tasks.kyu8 import example
+from tasks.kyu8 import abbrev_name
 
 
-def run_example():
-    """ToDo"""
-    first = input("a: ")
-    second = input("b: ")
-    print(example(first, second))
+def run_abbrev_name():
+    """
+    Runs abbrev_name function.
+    """
+    name = ''
```

Comment:
You can drop name='', because its useless line

Reply:
done